mandos-go/json/model/valueCheck.go

```diff
@@ -69,23 +73,25 @@ func (jcbytes JSONCheckBytes) Check(other []byte) bool {
 // Values are checked for equality.
 // "*" allows all values.
 type JSONCheckBigInt struct {
-	Value    *big.Int
-	IsStar   bool
-	Original string
+	Value       *big.Int
+	IsStar      bool
+	Original    string
+	Unspecified bool
 }
 
-// JSONCheckBigIntDefault yields JSONCheckBigInt default "*" value.
-func JSONCheckBigIntDefault() JSONCheckBigInt {
+// JSONCheckBigIntUnspecified yields JSONCheckBigInt default "*" value.
+func JSONCheckBigIntUnspecified() JSONCheckBigInt {
 	return JSONCheckBigInt{
-		Value:    nil,
-		IsStar:   true,
-		Original: "",
+		Value:       big.NewInt(0),
+		IsStar:      false,
+		Original:    "",
+		Unspecified: true,
```

Comment:
Optionally add the following:
```
type JSONCheckValue struct {
    IsStar bool
    Original string
    Unspecified bool
}
```
and embed it into structs such as `JSONCheckBigInt`.